modules/k8s/service.go

```diff
@@ -68,7 +68,15 @@ func WaitUntilServiceAvailable(t *testing.T, options *KubectlOptions, serviceNam
 			if err != nil {
 				return "", err
 			}
-			if !IsServiceAvailable(service) {
+
+			isMinikube, err := IsMinikubeE(t, options)
+			if err != nil {
+				return "", err
+			}
+
+			// For minikube, all services will be available immediately so we only do the check if we are not on
```

Comment:
Why are they available immediately? Is it not async / event-based under the hood?

Reply:
`NodePort` and `ClusterIP` are available immediately at creation time because all the information/control is available to the master components.